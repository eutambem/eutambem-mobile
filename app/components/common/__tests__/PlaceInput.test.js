import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import sinon from 'sinon';

import PlaceInput from '../PlaceInput';

const props = {
  placeScreenTitle: 'Inform the place',
  types: ['establishment'],
  onValueChange: sinon.spy(),
  navigation: {
    navigate: sinon.spy(),
    goBack: sinon.spy(),
  },
  placeholder: 'Place',
};

describe('PlaceInput', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<PlaceInput {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
