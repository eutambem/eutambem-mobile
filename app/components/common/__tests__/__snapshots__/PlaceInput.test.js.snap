// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceInput renders correctly 1`] = `
<TextInput
  multiline={false}
  navigation={
    Object {
      "goBack": [Function],
      "navigate": [Function],
    }
  }
  onFocus={[Function]}
  onValueChange={[Function]}
  placeScreenTitle="Inform the place"
  placeholder="Place"
  required={false}
  types={
    Array [
      "establishment",
    ]
  }
/>
`;
